src/index.tsx

```diff
@@ -172,8 +178,20 @@ function Dashboard() {
             .then(() => setExportLoading(false))
     }
 
+    const shouldRenderBackButton = useMediaQuery('(max-width:600px)')
```

Comment:
🤔What about webpackEnv?

Reply:
There's no choice about `mobile` in webpackEnv...

Reply:
If you use mediaquery here, and someone use it on a Android tablet or iPad, they will find they have no way to exit the screen because width is bigger than 600px

Reply:
:awsl: so let's add one def to indicate if it is built for desktop...

Reply:
!geckoView && !WKWebKit is enough